.php_cs.dist

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Should this file be excluded in .gitattributes?

Reply:
Fixed